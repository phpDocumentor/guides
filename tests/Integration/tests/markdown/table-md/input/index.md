# Table Markdown

| Name       | Age | City         |
|------------|-----|--------------|
| John Doe   | 29  | New York     |
| Jane Smith | 34  | San Francisco|
| Sam Green  | 22  | Boston       |
