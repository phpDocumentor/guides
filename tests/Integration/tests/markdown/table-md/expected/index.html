<!-- content start -->

<div class="section" id="table-markdown">
            <h1>Table Markdown</h1>
            <table>
                <thead>
            <tr>
                            <th>Name</th>
                            <th>Age</th>
                            <th>City</th>
                    </tr>
        </thead>
    <tbody>
    <tr>
            <td>John Doe</td>
            <td>29</td>
            <td>New York</td>
    </tr>
    <tr>
            <td>Jane Smith</td>
            <td>34</td>
            <td>San Francisco</td>
    </tr>
    <tr>
            <td>Sam Green</td>
            <td>22</td>
            <td>Boston</td>
    </tr>
</tbody>
</table>
    </div>
<!-- content end -->
